e2e/content-services/directives/favourite-directive.e2e.ts

```diff
@@ -0,0 +1,229 @@
+import { browser } from 'protractor';
+import { FileModel } from '../../models/ACS/fileModel';
+import resources = require('../../util/resources');
+import { NavigationBarPage } from '../../pages/adf/navigationBarPage';
+import { CustomSources } from '../../pages/adf/demo-shell/customSourcesPage';
+import { TrashcanPage } from '../../pages/adf/trashcanPage';
+
+describe('Favourite directive', function () {
+
+    const loginPage = new LoginPage();
+    const contentServicesPage = new ContentServicesPage();
+    const navigationBarPage = new NavigationBarPage();
+    const acsUser = new AcsUserModel();
+    const customSourcesPage = new CustomSources();
+    const trashcanPage = new TrashcanPage();
+    const contentListPage = contentServicesPage.getDocumentList();
+    const contentNodeSelector = new ContentNodeSelectorDialogPage();
+    this.alfrescoJsApi = new AlfrescoApi({
+        provider: 'ECM',
+        hostEcm: browser.params.testConfig.adf.url
+    });
+    const pdfFile = new FileModel({
+        'name': resources.Files.ADF_DOCUMENTS.PDF.file_name,
+        'location': resources.Files.ADF_DOCUMENTS.PDF.file_location
+    });
+
+    const uploadActions = new UploadActions(this.alfrescoJsApi);
+    let testFolder1, testFolder2, testFolder3, testFolder4, testFile;
+
+    beforeAll(async (done) => {
+
+        await this.alfrescoJsApi.login(browser.params.testConfig.adf.adminEmail, browser.params.testConfig.adf.adminPassword);
+
+        await this.alfrescoJsApi.core.peopleApi.addPerson(acsUser);
+
+        await this.alfrescoJsApi.login(acsUser.id, acsUser.password);
+
+        testFolder1 = await uploadActions.createFolder(StringUtil.generateRandomString(5), '-my-');
+
+        testFolder2 = await uploadActions.createFolder(StringUtil.generateRandomString(5), '-my-');
+
+        testFolder3 = await uploadActions.createFolder(StringUtil.generateRandomString(5), '-my-');
+
+        testFolder4 = await uploadActions.createFolder(StringUtil.generateRandomString(5), '-my-');
+
+        testFile = await uploadActions.uploadFile(pdfFile.location, pdfFile.name, '-my-');
+
+        await loginPage.loginToContentServicesUsingUserModel(acsUser);
+
+        contentServicesPage.goToDocumentList();
+
+        done();
+    });
+
+    afterAll(async (done) => {
+        await this.alfrescoJsApi.login(browser.params.testConfig.adf.adminEmail, browser.params.testConfig.adf.adminPassword);
+        await uploadActions.deleteFileOrFolder(testFolder1.entry.id);
+        await uploadActions.deleteFileOrFolder(testFolder2.entry.id);
+        await uploadActions.deleteFileOrFolder(testFolder3.entry.id);
+        await uploadActions.deleteFileOrFolder(testFolder4.entry.id);
+        done();
+    });
+
+    beforeEach(async (done) => {
+        navigationBarPage.clickContentServicesButton();
+        contentServicesPage.getContentList().dataTablePage().waitTillContentLoaded();
+        done();
+    });
+
+    it('[C260247] Favorite a file', async () => {
```

Comment:
Can we change the name of the tests to follow the convention ? It should be something like "Should be able to mark a file as favourite". The name needs to be changed also in the test case.